api/app/src/test/kotlin/packit/integration/controllers/ConfigurationControllerTest.kt

```diff
@@ -0,0 +1,13 @@
+package packit.integration.controllers
+
+import org.junit.jupiter.api.Test
+import packit.integration.IntegrationTest
+
+class ConfigurationControllerTest : IntegrationTest() {
+    @Test
+    fun `can get config`() {
+        val result = restTemplate.getForEntity("/configuration/logo", String::class.java)
+
+        assertSuccess(result)
```

Comment:
Might as well check we get expected content

Reply:
This depends on the env vars, so have updated the test env vars properties file for this purpose